site/docs/components/alerts.md

```diff
@@ -1,118 +1,84 @@
+</div>
 
-### Dismissing
-
-Using the alert JavaScript plugin, it's possible to dismiss any alert inline. Here's how:
-
-- Be sure you've loaded the alert plugin, or the compiled Bootstrap JavaScript.
-- If you're building our JavaScript from source, it [requires `util.js`]({{ site.baseurl }}/docs/getting-started/javascript/#util). The compiled version includes this.
-- Add a dismiss button and the `.alert-dismissible` class, which adds extra padding to the right of the alert and positions the `.close` button.
-- On the dismiss button, add the `data-dismiss="alert"` attribute, which triggers the JavaScript functionality. Be sure to use the `<button>` element with it for proper behavior across all devices.
-- To animate alerts when dismissing them, be sure to add the `.fade` and `.show` classes.
-
-You can see this in action with a live demo:
-
-{% capture example %}
-<div class="alert alert-warning alert-dismissible fade show" role="alert">
-  <strong>Holy guacamole!</strong> You should check in on some of those fields below.
+<div class="alert-heading alert-msg-info" role="alert">
   <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-    <span aria-hidden="true">&times;</span>
+    <svg class="alert-close"><use xlink:href="#close"/></svg>
   </button>
+  <h4 class="font-family-bold"><svg><use xlink:href="#cog"/></svg> Well done!</h4>
+  <p class="mb-0">Aww yeah, you successfully read this important alert message with an icon. This example text is going to run a bit longer so that you can see how spacing within an alert works with this kind of content.</p>
 </div>
+
 {% endcapture %}
 {% include example.html content=example %}
 
-## JavaScript behavior
-
-### Triggers
-
-Enable dismissal of an alert via JavaScript:
-
-{% highlight js %}
-$('.alert').alert()
-{% endhighlight %}
-
-Or with `data` attributes on a button **within the alert**, as demonstrated above:
-
-{% highlight html %}
-<button type="button" class="close" data-dismiss="alert" aria-label="Close">
-  <span aria-hidden="true">&times;</span>
-</button>
-{% endhighlight %}
 
-Note that closing an alert will remove it from the DOM.
+<!-- Bootstrap 4 -->
 
-### Methods
+***
 
-| Method | Description |
-| --- | --- |
-| `$().alert()` | Makes an alert listen for click events on descendant elements which have the `data-dismiss="alert"` attribute. (Not necessary when using the data-api's auto-initialization.) |
-| `$().alert('close')` | Closes an alert by removing it from the DOM. If the `.fade` and `.show` classes are present on the element, the alert will fade out before it is removed. |
-| `$().alert('dispose')` | Destroys an element's alert. |
+## System Extras
+​
+Settings inherited from forking Bootstrap 4 and custom designs not part of the design system, yet could be beneficial in the future.
```

Comment:
flo-scss is meant to be agnostic of any individual "design system"; what we are documenting here is a css api, not the specifics of any one design. Bootstrap already gives us very mature and well vetted documentation put together by over a thousand contributors and read and understood by hundreds of thousands of developers. If design has a need to document a specific system, it should be kept closer to the design system itself and separate from the implementation details. These changes not only muddle the separation of concerns between design and implementation, but also break the known workflows of other devs already familiar with bootstrap.

Reply:
What is your specific recommended action so we don't go round and round on this? 

The "design system" referenced above is in reflection to the brand design system. I can add this for clarification separating the concept of what is a design system form the visual approved branding.

Or is simply the mentioning of Bootstrap 4 the concern?  Documenting the forking from Bootstrap 4 goal is to provide a developer a point of reference to how the system has more possible layout variants outside the approved visual design system.

Reply:
I think that is where the disconnect is. I originally thought flo-scss was where the flo design system would be documented. This discussion should be moved to a video chat or meeting because I think there is still a misunderstanding of this project and where different aspects of the flo design brand will live.

Reply:
also the link you shared shows examples of themes that only have color changes, maybe a font change. There's nothing that changes the look and functionality of bootstrap. Flo brand does not look like bootstrap with a color change. So how do we deal with that? Where does the code that will _change_ that live if not here? In the webapp?

Reply:
Yes, feel free to set up a discussion sometime this week. My calendar is wide open this week.

I think this bootswatch link does provide several demonstrations of significant changes to look and feel by adjusting not only colors and fonts, but also borders, radii, sizing, spacing, etc. These are the true "atoms" from which the visual treatment is composed and are generally well represented in `_variables.scss`.  See https://bootstrap.build/app for an interactive example of what can be accomplished only by modifying stock variables.  I'm not sure what is meant by "functionality" here, as most interaction will be deferred to consuming applications and/or libraries. It is still true that we can remove references to bootstrap javascript and jquery, since we don't maintain or publish that, but in theory a php-based hackathon project should be able to import flo-scss and bootstrap.js and expect it to work as documented by bootstrap.